boilerplates/fe-angular/README.md

```diff
@@ -0,0 +1,82 @@
+# fe-angular
+
+Angular quickstarter project
+
+## Purpose of this quickstarter
+ 
+ This quick starter gives you a new Angular project. This project contains basic setup for docker, jenkins, sonarqube and OpenShift. 
+
+## What files / architecture is generated?
+ ```
+.
+├── Jenkinsfile
```

Comment:
I think we should add a comment next to thow - e.g. JenkinsFile - this is the file that controls the build

Reply:
added